uvicorn/config.py

```diff
@@ -395,37 +395,63 @@ def setup_event_loop(self) -> None:
             loop_setup()
 
     def bind_socket(self) -> socket.socket:
-        family = socket.AF_INET
-        addr_format = "%s://%s:%d"
+        logger_args: List[Union[str, int]]
```

Comment:
just needed to add this post-merge with master @Kludex 
the rest stayed the same

Reply:
So  I removed the hold button and feel free to merge if still ok with this !

Reply:
I find it so weird that the approval is not removed after a commit is pushed. I don't understand the reason behind it.

Anyway, LGTM! :)